packages/pluggableWidgets/web-view-native/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "web-view-native",
   "widgetName": "WebView",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
What's the reason this version is out of sync with the other widgets updated in this PR?

Reply:
Explained